opentracing-api/src/main/java/io/opentracing/propagation/Adapters.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016-2017 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.propagation;
+
+import java.io.IOException;
+
+public final class Adapters {
```

Comment:
I do not see a real benefit of having this class. Simply calling the appropriate constructor of BinaryAdapter in a call to inject or extract should do.

Reply:
The idea is to put other formats here, so we can abstract the actual implementations, so we don't have to expose `BinaryAdapter` and others.